src/components/PaginationFooter/types.ts

```diff
@@ -0,0 +1,53 @@
+import type { stardust } from '@nebula.js/stardust';
+import type { ExtendedTheme } from '../../hooks/use-extended-theme/types';
+
+export enum ButtonTypes {
+  FIRST_PAGE = 'FirstPage',
+  LAST_PAGE = 'LastPage',
+  NEXT_PAGE = 'NextPage',
+  PREVIOUS_PAGE = 'PreviousPage',
+}
+
+export interface FooterWrapperProps {
+  children: JSX.Element;
+  footerContainer?: HTMLElement;
+  paginationNeeded?: boolean;
+  theme: ExtendedTheme;
+  interactions: stardust.Interactions;
+}
+
+export interface FooterStyle {
+  borderColor: string;
+  color: string;
+  disabledColor: string;
+  iconColor?: string;
+  background?: string;
+}
+
+export interface ExtendedTranslator extends stardust.Translator {
```

Comment:
this one should be needed for the header as well I guess?

Reply:
I think we dont even need it since nebula already have it baked in for us: 
https://github.com/qlik-oss/nebula.js/blob/38b80cb3a5af40e9458ae966db85c9749e4ccb99/apis/locale/src/translator.js#L20-L25

Reply:
Good find. Will look into this and remove if possible